### PR TITLE
BaSP-MR-53: all methods for trainers

### PR DESCRIPTION
**Pull request by GaloDurante**
I worked on the methods for trainers and the CSS.

### CODE REVIEW

File: src/Components/Trainers/index.js [hunk 1]
```diff
@@ -1,9 +1,369 @@
...
+        </button>
+        <table>
+          <thead>
+            <th className={`${styles.head}`}>Name</th>
+            <th>Dni</th>
+            <th>Email</th>
+            <th>Phone</th>
+            <th>City</th>
+            <th>Salary</th>
+            {/* <th className={`${styles.row} ${styles.head}`}>Status</th> */}
```
Comment (Eugeniagassman) [inline src/Components/Trainers/index.js]: You should delete this comment

File: src/Components/Trainers/index.js [hunk 2]
```diff
@@ -1,9 +1,369 @@
...
+                <tr key={trainer._id}>
+                  <td className={styles.row}>
+                    {trainer.firstName} {trainer.lastName}
+                  </td>
+                  <td className={styles.row}>{trainer.dni}</td>
+                  <td className={styles.row}>{trainer.email}</td>
+                  <td className={styles.row}>{trainer.phone}</td>
+                  <td className={styles.row}>{trainer.city}</td>
+                  <td className={styles.row}>{trainer.salary}</td>
+                  {/* <td className={styles.row}>{trainer.isActive}</td> */}
```
Comment (Eugeniagassman) [inline src/Components/Trainers/index.js]: same here

File: src/Components/Trainers/index.js [hunk 3]
```diff
@@ -1,9 +1,364 @@
...
+        setFormValue({
+          firstName: '',
+          lastName: '',
+          dni: '',
+          phone: '',
+          email: '',
+          city: '',
+          password: '',
+          salary: '',
+          isActive: true
```
Comment (Eugeniagassman) [inline src/Components/Trainers/index.js]: this should be a selector to change the state of isActive to either true or false

File: src/Components/Trainers/index.js [hunk 4]
```diff
@@ -1,9 +1,364 @@
...
+    setFormValue({
+      firstName: data.firstName,
+      lastName: data.lastName,
+      dni: data.dni,
+      phone: data.phone,
+      email: data.email,
+      city: data.city,
+      password: data.password,
+      salary: data.salary,
+      isActive: true
```
Comment (Eugeniagassman) [inline src/Components/Trainers/index.js]: here you should also be able to edit the isActive status.

**Review by Eugeniagassman — APPROVED** [review]
looks good to me!

**Review by leimeter-joaquin — APPROVED** [review]

**Review by leimeter-joaquin — APPROVED** [review]

**Comment by Flor-Farruggia** [comment]
Hi, you have done a really nice job, you only have some validation problems, but are minor priority, especially when you don't actually have to do validation this week, so i leave you a ticket for you have in considaration for when you have to develop the valitions, but in general it was a really amazing job and the flow work perfect, congrats!

**Pull request merged**